vertica_python/vertica/column.py

```diff
@@ -90,7 +90,10 @@ def __init__(self, col):
         self.props = ColumnTuple(col['name'], col['data_type_oid'], None,
                                  col['data_type_size'], None, None, None)
 
-        self.converter = self.DATA_TYPE_CONVERSIONS[col['data_type_oid']][1]
+        try:
+            self.converter = self.DATA_TYPE_CONVERSIONS[col['data_type_oid']][1]
```

Comment:
Could you also do this like:
```
self.converter = self.DATA_TYPE_CONVERSIONS.get(col['data_type_oid'], 0)[1]
```
?

Might be a bit more concise, but other than that, looks good.